ors-engine/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/builders/HeavyVehicleGraphStorageBuilder.java

```diff
@@ -179,22 +178,13 @@ public void processWay(ReaderWay way) {
                 }
 
                 if (motorVehicleHgvValues.contains(key)) {
-
-                    // TODO: the following implementation does not pick up access:destination
```

Comment:
Is `access=destination` picked up by the new implementation? 
If not, I think we should keep the comment?

Reply:
According to @aoles, access destination appears there only because a few years ago there was some idea to address the routing on access destination edges through this edge filter, but this never worked properly.